library/publishing.gradle

```diff
@@ -1,19 +1,13 @@
-def artifactory_repo_key = System.getenv('ARTIFACTORY_REPO_KEY') != null ? System.getenv('ARTIFACTORY_REPO_KEY') : 'libs-release-local'
-def artifactory_username = System.getenv('ARTIFACTORY_USERNAME')
-def artifactory_password = System.getenv('ARTIFACTORY_PASSWORD')
+def garProjectId = System.getenv("GOOGLE_CLOUD_PROJECT")
+def garLocation = System.getenv("GAR_LOCATION")
+def garRepoId = System.getenv("GAR_REPOSITORY_ID")
+
 
 java {
     // Builds sources into the published package as part of the 'assemble' task.
     withSourcesJar()
 }
 
-gradle.taskGraph.whenReady { taskGraph ->
```

Comment:
I think this is artifactory-specific and so can be removed entirely instead of updated.  My test-run of publishing seemed to work without it, but if you know otherwise, let me know!